Dockerfile

```diff
@@ -12,9 +12,9 @@ COPY go.sum go.sum
 RUN go mod download
```

Comment:
Let's remove this line to make the build more efficient as we already have all the dependencies in `vendor`.